py/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 This release requires Python 3.9 or later. There are no intentionally breaking changes included in this release, but we stopped testing against Python 3.8.
 
+- Normalize event timestamps before validating them, fixing cases where Relay would drop valid events with reason "invalid_transaction". ([#2851](https://github.com/getsentry/relay/issues/2851))
```

Comment:
Could you , please fix the record above to start with `-` and remove space between them?